content/blog/pulkomandy/2020-04-28-haiku_activity_report_april_2020.md

```diff
@@ -25,7 +25,7 @@ next one!
 <h3>Outside the source tree</h3>
 
 To celebrate the new beta release, 3dEyes has been working on porting GTK
-to Haiku. While of course our main focus remain on native applications using the
+to Haiku. While of course our main focus remains on native applications using the
 BeAPI, it's still a great help for new users if they can find some of their well
 known apps at first. The port is nowhere near ready yet, but we've seen some
 screenshots already...
```

Comment:
Our forum software seems to have an issue display elipses. It shows "& mldr;" instead of the dots. Better use 3 dots "..." instead.

Reply:
Those are 3 dots. :)

Reply:
It's the UTF elipsis character that's still displayed (in Otter and Web+ at least) as "& mldr". Three separate dot characters would work around that. The best solution would of course be to make the website correctly display the elipsis...

Reply:
No, I mean that the source for this post is already using 3 dots on line 31 above.

Reply:
Weird. It's still "& mldr;" on the site... Some 'helpful' auto-conversion somewhere?